src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala

```diff
@@ -26,4 +26,37 @@ class AzulZuluMigrations {
       Version("java", "11.0.5-zulu", "https://cdn.azul.com/zulu/bin/zulu11.35.15-ca-jdk11.0.5-macosx_x64.tar.gz", MacOSX, Some(Zulu))
     ).validate().insert()
   }
+
+  @ChangeSet(order = "003", id = "003-add_zulu_8_0_232", author = "poad")
+  def migrate003(implicit db: MongoDatabase) = {
+    Seq(Linux64, Windows, MacOSX).foreach(platform => removeVersion(candidate = "java", version = "8.0.232-zulu", platform))
```

Comment:
Probably don't need this here as it is already at the end of the migration.

Reply:
Thx! Removed this line, and modify `"003-add_zulu_8_0_232"` to `"003-add_zulu_8_0_242"` at line 30.